packages/nimble-components/src/table-column/number-text/models/tests/manually-translated-unit-scale-formatter.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { ManuallyTranslatedUnitScaleFormatter } from '../manually-translated-unit-scale-formatter';
+import { UnitPrefix } from '../unit-prefix';
+import { UnitTranslation } from '../unit-translation';
+
+class TestManuallyTranslatedUnitScaleFormatter extends ManuallyTranslatedUnitScaleFormatter {
+    protected override getUnitTranslations(): Map<string, UnitTranslation> {
+        const translations = new Map<string, UnitTranslation>();
+        translations.set(
+            'en',
+            new UnitTranslation('en-singular', 'en-plural', 'en-abbrev')
+        );
+        translations.set(
+            'fr',
+            new UnitTranslation('fr-singular', 'fr-plural', 'fr-abbrev')
+        );
+        return translations;
+    }
+
+    protected override getSupportedPrefixes(): UnitPrefix[] {
+        return [new UnitPrefix(5, '5.'), new UnitPrefix(10, '10.')];
+    }
+}
+
+describe('ManuallyTranslatedUnitScaleFormatter', () => {
```

Comment:
Would it be worthwhile to add a test that you can provide just the language portion of the locale but not the region portion? e.g (`fr` instead of `fr-FR`)